homeassistant/components/music_assistant/services.yaml

```diff
@@ -0,0 +1,254 @@
+          integration: music_assistant
+    name:
+      required: true
+      example: "We Are The Champions"
+      selector:
+        text:
+    media_type:
+      required: false
+      example: "playlist"
+      selector:
+        select:
+          multiple: true
+          translation_key: media_type
+          options:
+            - artist
+            - album
+            - playlist
+            - track
+            - radio
+    artist:
+      required: false
+      example: "Queen"
+      selector:
+        text:
+    album:
+      required: false
+      example: "News of the world"
+      selector:
+        text:
+    limit:
+      required: false
+      advanced: true
+      example: 25
+      default: 5
+      selector:
+        number:
+          min: 1
+          max: 100
+          step: 1
+    library_only:
+      required: false
+      example: "true"
+      default: false
+      selector:
+        boolean:
+
+get_library:
+  fields:
+    config_entry_id:
+      required: true
+      selector:
+        config_entry:
+          integration: music_assistant
+    media_type:
+      required: true
+      example: "playlist"
+      selector:
+        select:
+          translation_key: media_type
+          options:
+            - artist
+            - album
+            - playlist
+            - track
+            - radio
+    favorite:
+      required: false
+      example: "true"
+      default: false
+      selector:
+        boolean:
+    search:
+      required: false
+      example: "We Are The Champions"
+      selector:
+        text:
+    limit:
+      required: false
+      advanced: true
+      example: 25
+      default: 25
+      selector:
+        number:
+          min: 1
+          max: 500
+          step: 1
+    offset:
+      required: false
+      advanced: true
+      example: 25
+      default: 0
+      selector:
+        number:
+          min: 1
+          max: 1000000
+          step: 1
+    order_by:
+      required: false
+      example: "playlist"
+      selector:
+        select:
+          translation_key: order_by
+          options:
+            - name
+            - name_desc
+            - sort_name
+            - sort_name_desc
+            - timestamp_added
+            - timestamp_added_desc
+            - last_played
+            - last_played_desc
+            - play_count
+            - play_count_desc
+            - year
+            - year_desc
+            - position
+            - position_desc
+            - artist_name
+            - artist_name_desc
+            - random
+            - random_play_count
+    album_type:
+      required: false
+      example: "single"
+      selector:
+        select:
+          multiple: true
+          translation_key: album_type
+          options:
+            - album
+            - single
+            - compilation
+            - ep
+            - unknown
+    album_artists_only:
+      required: false
+      example: "true"
+      default: false
```

Comment:
Note: this might be broken. If I'm correct, this selector will now always return false (even if it was set to `true` in the UI).